README.md

```diff
@@ -96,7 +79,38 @@ Appledoc for the SDK can be found [here](http://cocoadocs.org/docsets/TMTumblrSD
 If you install the Tumblr SDK using CocoaPods, the docset is automatically added
 to Xcode for you.
 
-## Authentication
+## Connecting to Tumblr
+
+There are two ways for your application to connect with Tumblr, each with different benefits.  You are free to choose the method that you prefer based on the needs of your application.  
+
+### API client
+
+The first method is the API Client, a class that facilitates direct communication with Tumblr via the [Tumblr API](http://www.tumblr.com/docs/en/api/v2).  The benefits of using the API Client include the ability to better control the user experience and keep users in your app.  A typical use case for the API client is an application that allows a user to share a photo to multiple social networks at the same time.  In this case, after allowing a user to select a photo, you might present them with a list of toggles representing the different networks that would receive the photo.  Once they confirm, and assuming they selected Tumblr, you would use the API client to post the image to the user's Tumblr.  Use of the API Client requires [authentication](#authentication) from the user for certain routes which you may facilitate through the `TMTumblrAuthenticator` class.  
+
+Please view the [API documentation](http://www.tumblr.com/docs/en/api/v2) for
+full usage instructions.
+
+There are two ways of retrieving data from the API:
+
+``` objectivec
+// `void` methods for immediate requests, preferable when the caller does not need a reference to an actual request object:
+
+[[TMAPIClient sharedInstance] userInfo:^(id result, NSError *error) {
+    if (!error)
+        NSLog(@"Got some user info");
+}];
+
+// Methods that return configured, signed `JXHTTPOperation` instances and require the client to explicitly send the request separately.
+
+JXHTTPOperation *likesRequest = [[TMAPIClient sharedInstance] likesRequest:@"bryan" parameters:nil];
+```
+
+The API client is built on top of the
+[JXHTTP](https://github.com/jstn/JXHTTP) networking library. If you're only
```

Comment:
This link is broken. I believe this change should incorporate the fix (https://github.com/tumblr/TMTumblrSDK/pull/117).